### ISSUE TITLE
Revisit Cook and Food

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#11399]
As of 8.0 with the addition of many new plants and the fact that anyone can plant and harvest food, cooks - a profession that already was very unbalanced in 0.7 - has become pretty much useless.

We would need to take a look on how to discourage players from doing their food stuff themselves and allow cooks to have an impact that can't be neglected.

I would propose the following things, and others as well as @elfl0rd will probably jump in and add to that:

1. Late game, already in 7.x was a phase in which cooks became pretty irrelevant due to the lack of calorie consumption when using machinery. In my opinion the usage of any machinery should also take calories over time or on action, just at a reduced rate. I assume someone steering a excavator all day get's exhausted. At least they look like that when i come by a construction site. I would not want to do do their job.

2. Make farming useful in some way, giving real benefits. For example players that are not having the farmers profession and skills could have a high chance in failing to grow plants and harvest them - there are possibly way better solutions on how to do this, but this could help. While i'm totally able to harvest some huckleberries on a wild bush i'm probably not able to get as many of them as i could get at a bush artifically placed and cared about by someone that is skilled. And planting things i would probably not get that good yield or quality of things until i learned how i can actually treat those plants right. In any case it needs to be rebalanced that people without farmer / cooking skills can not self-sustain themselves just by planting their own things.

3. Give a noticeable incentive to use cooks goods. Yes, it's something min-max players probably do, but the average player does not really need good food - they push up their housing value to a max and have enough points - permanently, without any need to reinvest. Food then is just for not getting hungry. Just like eating fast-food all day. A balance between housing and food would be something to consider. Something i'm searching for my server is a limiter, that does not allow you to get more housing points than you have food points. That might not be a good solution as of game design, but it would fix the problem and make people care about what they eat.

Also see my suggestion from last year: https://github.com/StrangeLoopGames/EcoSuggestions/issues/592

Posted on request by John.

**Comment by elfl0rd** [comment]
I agree. Also exploit with work boots+work backpack+self improvements need to be fixed asap.

I would also doubles or triples your total calorie intake.

About farming.
1) no (or really low) harvest seeds from plants. You need a farmer, to make seeds from harvested plants.
2) More resources consumption for cooking. In the current form, for example, a steam tractor is almost not used. (Large fields? Pffff.)
2.1) refactor steam tractor inventory - https://github.com/StrangeLoopGames/EcoSuggestions/issues/430
3) More using of fertilizers, soils are depleted faster, fertilizers are weaker and require more resources to create.
4) More calories consumption while player move not at road, and in high grass\bush\debris. (more penalty = more calories consumption).
4.1) Not bad if default dirt speed is 0.9 (This will make the road preferable to bare ground). (but stones still 1.0)
5) Improve fishing, please - more mid and late game recipes from fish, and more fish usability (Sushi? Fish oil?). Current - it's only low quality calories.

**Comment by Conchipedia** [comment]
** More calories consumption while player move not at road, and in high grass\bush\debris. (more penalty = more calories consumption).** this would effect Hunter badly as they are never normally on a road Hunting (they have enough problems with Bushes slowing them down as it is) Maybe Hunting can decrease the effect of Grass\Bush and Debris? 🏹

**Comment by elfl0rd** [comment]
nice addition.

**Comment by redwyre** [comment]
Food degradation over time would help the farmer be more useful as people can't just stockpile food, which would in turn give chefs a more continued role

**Comment by elfl0rd** [comment]
Good idea too, we already have "rotting" for garbage, why not make it for food that isn't in the icebox\fridge?

**Comment by elfl0rd** [comment]
Also about cutting edge cooking, - requiere high price table, 3 tier room... But... Useless.

**Comment by elfl0rd** [comment]
Also i think not bad if we keep https://github.com/StrangeLoopGames/EcoSuggestions/issues/724 as part of this.

**Comment by DragonSupremacy** [comment]
While we're at it, the highlighted portion is entirely imbalanced as well. 80% overall calorie efficiency reduction when it was just 25% pre-8.0 is just entirely too much, and likely why survivalist 7 + work backpack + work boots mean 0 calorie usage.

Proposal: Overall calorie reduction to range from 10 to 25% in steps of 2.5%

![20190224233623_1](https://user-images.githubusercontent.com/841990/53308080-9e19f400-389e-11e9-98b9-bea937942f7c.jpg)

**Comment by MeirionHughes** [comment]
I'd propose also a reduction of the food benefit (skill points) if you stack the same food in your stomach. i.e. after 3 apply a 5% reduction (accumulative) per additional item (past 3). This would have the goal of encouraging a more varied diet.
